mlrun/feature_store/retrieval/spark_merger.py

```diff
@@ -88,6 +89,10 @@ def _generate_vector(
         # join the feature data frames
         self.merge(entity_rows, entity_timestamp_column, feature_sets, dfs)
 
+        # filter joined data frame
```

Comment:
you added a `#` header comment here but not elsewhere. Let's try to write a good looking code anywhere even if it's not there in the first place :)